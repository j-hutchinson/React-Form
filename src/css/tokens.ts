import { css } from 'styled-components';

export const minWidth = '800px';

export const PanelStyles = css`
    background-color: white;
    border: 1px solid black;
    font-size: 12px;
    box-sizing: border-box;
    text-decoration: none;
    color: black;
    padding: 20px;
`;
