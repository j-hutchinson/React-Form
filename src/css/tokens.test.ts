import * as tokens from './tokens';

describe('tokens', () => {
    for (let key in tokens) {
        test(key, () => {
            expect.assertions(1);
            expect(tokens[key]).toMatchSnapshot();
        });
    }
});
