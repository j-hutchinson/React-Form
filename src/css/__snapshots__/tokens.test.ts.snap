// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tokens PanelStyles 1`] = `
Array [
  "
    background-color: white;
    border: 1px solid black;
    font-size: 12px;
    box-sizing: border-box;
    text-decoration: none;
    color: black;
    padding: 20px;
",
]
`;

exports[`tokens minWidth 1`] = `"800px"`;
